### ISSUE TITLE
Lovelace as the second parameter for getLiquidityPools() returns nothing

### ISSUE THREAD

**Issue by Sluder** [IndigoProtocol/dexter#16]
Found using Blockfrost as the data provider, but test using API as well

**Comment by dpbeaumont** [comment]
Also occurs when using lovelace as the first parameter.
Replication steps are: 

```
 const request = dexter.newFetchRequest().forDexs([WingRiders.name]);

const poolsForToken1 = await request.getLiquidityPools('lovelace', new Asset(arbitrageToken?.policyId, arbitrageToken?.assetName, arbitrageToken?.decimals));

const poolsForToken2 = await request.getLiquidityPools(new Asset(arbitrageToken?.policyId, arbitrageToken?.assetName, arbitrageToken?.decimals), 'lovelace');

```

Occasionally the variable for the second call (poolsForToken2) is empty when looping.

**Comment by Sluder** [comment]
@dpbeaumont Closing this issue as this was resolved with https://github.com/IndigoProtocol/dexter/pull/24 (tested & was unable to replicate).
There are a lot of Blockfrost requests needed for WingRiders specifically, and Im betting you were getting rate limited on the second call which failed silently & returned no pools. https://github.com/IndigoProtocol/dexter/pull/24 allows those failures to bubble up so the API fallback can kick in

**Comment by dpbeaumont** [comment]
> @dpbeaumont Closing this issue as this was resolved with #24 (tested & was unable to replicate). There are a lot of Blockfrost requests needed for WingRiders specifically, and Im betting you were getting rate limited on the second call which failed silently & returned no pools. #24 allows those failures to bubble up so the API fallback can kick in

Completely agree, I also think the second call is a rate limiting issue.
